src/cdk/database/serverless-pg.ts

```diff
@@ -0,0 +1,60 @@
+import { DatabaseClusterEngine, IClusterEngine, ServerlessCluster, ServerlessClusterProps } from "@aws-cdk/aws-rds"
+import * as core from "@aws-cdk/core"
+
+export interface SlsPgDbProps extends Omit<ServerlessClusterProps, "engine"> {
+  // defaults to AURORA_POSTGRESQL
+  engine?: IClusterEngine
+
+  dbName?: string | undefined
+}
+
+/**
+ * Quickest and easiest way to get a Postgres database.
+ * Uses Aurora Serverless.
+ * Generates secret.
+ * Recommend to use with https://www.npmjs.com/package/typeorm-aurora-data-api-driver
+ * and https://docs.aws.amazon.com/cdk/api/latest/docs/aws-rds-readme.html#data-api
+ */
+export class SlsPgDb extends ServerlessCluster {
+  dbName?: string | undefined
+
+  constructor(scope: core.Construct, id: string, { dbName, ...props }: SlsPgDbProps) {
+    const superProps = {
+      engine: props.engine || DatabaseClusterEngine.AURORA_POSTGRESQL,
```

Comment:
If this is a PG only class, shouldn't we disallow the user to change the engine? Also if the user changes to mysql, for instance, it will cause bugs, at least with the `SlsPgDb.makeDatabaseUrl` url and `SlsPgDb.getConnectionOptions` when using a local db.

Reply:
Ok it'll be forced to `AURORA_POSTGRESQL`